test/liblll/EndToEndTest.cpp

```diff
@@ -272,7 +272,7 @@ BOOST_AUTO_TEST_CASE(assembly_codecopy)
 			(seq
 				(lit 0x00 "abcdef")
 				(asm
-					0x06 0x16 0x20 codecopy
+					0x06 6 codesize sub 0x20 codecopy
```

Comment:
This test is WAY too fragile since it relies on the layout of the binary and I would propose to delete it.